static/index.html

```diff
@@ -38,8 +38,8 @@
                             <th width="150">Assignee</th>
                             <th width="150">Milestone</th>
                             <th width="100">State</th>
+                            <th width="100">Age (in hours)</th>
                             <th>Repo</th>
-                            <th>HTML</th>
```

Comment:
> Can the row be clickable but without having a column for it.?
I'm not sure about that. I will have to further check to have links without a specific column data